adapters/repos/db/inverted/bm25_searcher.go

```diff
@@ -342,140 +343,160 @@ func (b *BM25Searcher) getTopKHeap(limit int, results terms, averagePropLength f
 
 func (b *BM25Searcher) createTerm(N float64, filterDocIds helpers.AllowList, query string, propertyNames []string, propertyBoosts map[string]float32, duplicateTextBoost int, additionalExplanations bool) (term, map[uint64]int, error) {
 	termResult := term{queryTerm: query}
-	filteredDocIDs := sroar.NewBitmap() // to build the global n if there is a filter
 
-	allMsAndProps := make(AllMapPairsAndPropName, 0, len(propertyNames))
-	for _, propName := range propertyNames {
+	var eg errgroup.Group
```

Comment:
shouldn't we use here: `eg := enterrors.NewErrorGroupWrapper(m.logger)`?

Reply:
That wrapper is only present in 1.24+, the suggested merged for 1.24+ on the comment above already contain it.